corehq/apps/app_manager/suite_xml/features/mobile_ucr.py

```diff
@@ -449,6 +449,7 @@ def is_valid_mobile_select_filter_type(ui_filter):
     return isinstance(ui_filter, DynamicChoiceListFilter) or isinstance(ui_filter, ChoiceListFilter)
 
 
+@quickcache(['domain'])
```

Comment:
Depending on how confident you are that you're clearing it correctly, you may want to extend this cache beyond 5 minutes.